app/src/features/sessionBook/screens/DraftSessionScreen/components/SessionTrimmer/SessionTrimmer.tsx

```diff
@@ -1,74 +1,29 @@
-import React, { useCallback, useRef, useState } from "react";
+import React, { useCallback, useEffect, useRef, useState } from "react";
 import { useDispatch, useSelector } from "react-redux";
-import { getSessionRecordingEvents, getSessionRecordingMetaData } from "store/features/session-recording/selectors";
-import { RQButton } from "lib/design-system/components";
+import { getSessionRecordingEvents } from "store/features/session-recording/selectors";
 import "./sessionTrimmer.scss";
 import { EventType, eventWithTime } from "rrweb";
 import { fullSnapshotEvent, IncrementalSource } from "@rrweb/types";
 import { NetworkEventData, RQSessionEventType } from "@requestly/web-sdk";
 import { sessionRecordingActions } from "store/features/session-recording/slice";
 import { partition } from "lodash";
-import { Divider } from "antd";
 import { TrimHandle } from "./components/TrimHandle/TrimHandle";
 import { msToMinutesAndSeconds } from "utils/DateTimeUtils";
+import { useDebounce } from "hooks/useDebounce";
 
 interface SessionTrimmerProps {
   duration: number;
   sessionStartTime: number;
+  player: unknown;
```

Comment:
see if we can use a proper type for this except `unknown`. Either directly available from rrweb or create a composite type using the types defined [here](https://github.com/requestly/requestly/blob/master/app/src/features/sessionBook/types/index.ts)